setup.py

```diff
@@ -63,8 +64,8 @@ def install_requires():
 #####
 NAME = 'gordon-dns-gcp'
 PACKAGE_NAME = 'gordon_gcp'
-PACKAGES = ['gordon_gcp']
-META_PATH = os.path.join(PACKAGE_NAME, '__init__.py')
+PACKAGES = find_packages(where='src')
```

Comment:
To make sure I understand this correctly: the old version of this was listing only the top-level package (gordon_gcp), which caused setuptools to include all the package _modules_, but it excluded any _sub-packages_ like `plugins`?

I've looked through the setuptools docs and the [pypa tutorial](https://packaging.python.org/tutorials/distributing-packages/#packages), but this part is very vague.